gn3/db/wiki.py

```diff
@@ -9,6 +9,16 @@ class MissingDBDataException(Exception):
     """Error due to DB missing some data"""
 
 
+def _decode_dict(result: dict):
+    new_result = {}
+    for k, v in result.items():
```

Comment:
We have a pylint error here:

```
************* Module gn3.db.wiki
gn3/db/wiki.py:14:11: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)
```

I would suggest:

```suggestion
    for k, val in result.items():
```

Reply:
Done